### DISCUSSION TITLE
03/17 daily Spring

### DISCUSSION THREAD

**Discussion by revature-ives** [algoives/TestBankApp#35] [Daily Scrum]
**Jorge  Murillo**

**What I did yesterday**

I defined the entities and their attributes for the database

**Obstacles**

Ned more knowledge about database design 
how to create joins

**what I going to do today**

design and build database

**Comment by ClarkeWilliford** [comment]
I'm currently working on finishing putting placeholder code in the login screen. Then I'll move on to the Sign Up screen if I have time.

**Comment by RahmmaAwali** [comment]
Rahmma ,
what I did Yesterday.
I searched some video for my lunch screen. 


what I will do today 
I am currently referring to some videos from our lesson . at the end of today I'll put some videos on launch screen
